<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetSelect</class>
 <widget class="QGroupBox" name="TargetSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Search path:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CustomLineEdit" name="lineEditDirectory"/>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="toolButtonBrowse">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelWildcards">
     <property name="text">
      <string>File name wildcards (pipe separated):</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="CustomLineEdit" name="lineEditWildcards">
     <property name="text">
      <string>*.*</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelExcludes">
     <property name="text">
      <string>Exclude directories (pipe separated):</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="CustomLineEdit" name="lineEditExcludes"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLineEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomLineEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
